.github/workflows/ci.yml

```diff
@@ -18,3 +18,42 @@ jobs:
           cache: npm
       - run: npm ci
       - run: npm run test
+
+      - run: git fetch --unshallow
```

Comment:
Could you add a comment explaining why this is necessary?

Reply:
Will do